apps/step-function.json.j2

```diff
@@ -207,7 +207,9 @@
       "Parameters": {
         "JobDefinition": "{{ '${'+ snake_to_pascal_case(task['name']) + '}' }}",
         "JobName.$": "$.job_id",
-        "JobQueue": "${JobQueueArn}",
+        {% set name = job_spec['compute_environment']['name'] %}
+        {% set job_queue = name + 'JobQueueArn' if 'Default' != name else 'JobQueueArn' %}
+        "JobQueue": "{{ '${'+ job_queue +'}' }}",
```

Comment:
Note that this means the same compute environment will be used for all tasks within the job spec, which is fine for `SRG_GSLC`, but maybe we'll want to be able to specify AMI and instance types for an individual task, the same way we can specify `memory` and `vcpu` for an individual task? Not sure how difficult that would be to implement as part of this PR, so maybe not worth trying to support that until we need it.

Reply:
This is an interesting idea, I'm thinking about it.